package.json

```diff
@@ -0,0 +1,50 @@
+{
+  "name": "bandayd",
+  "version": "1.0.0",
+  "description": "This is a social media website for bands",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "react-dev": "webpack --mode development --watch",
+    "server-dev": "npx nodemon server/index.js"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/Blue-Ocean-Galley/Bandayd.git"
+  },
+  "author": "Aubery Bachant, Kaleb Lyda, Squall Leonhart, Sung Kim, Zach Kodjo, Bill Mehlenbacher",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/Blue-Ocean-Galley/Bandayd/issues"
+  },
+  "homepage": "https://github.com/Blue-Ocean-Galley/Bandayd#readme",
+  "dependencies": {
+    "@babel/core": "^7.13.1",
+    "axios": "^0.21.1",
+    "babel": "^6.23.0",
+    "babel-loader": "^8.2.2",
+    "body-parser": "^1.19.0",
+    "eslint": "^7.20.0",
+    "express": "^4.17.1",
+    "express-session": "^1.17.1",
+    "jquery": "^3.5.1",
+    "morgan": "^1.10.0",
+    "node": "^15.8.0",
+    "nodemon": "^2.0.7",
+    "path": "^0.12.7",
+    "react": "^17.0.1",
+    "react-dom": "^17.0.1"
+  },
+  "devDependencies": {
+    "@babel/core": "^7.13.1",
+    "@babel/preset-env": "^7.13.5",
+    "@babel/preset-react": "^7.12.13",
+    "@testing-library/jest-dom": "github:testing-library/jest-dom",
+    "@testing-library/react": "^11.2.5",
+    "babel-jest": "^26.6.3",
+    "jest": "^26.6.3",
+    "react-test-renderer": "^17.0.1",
+    "webpack": "^5.24.1",
+    "webpack-cli": "^4.5.0"
+  }
+}
```

Comment:
Can you remove these until we figure out what we're doing?
- body-parser
- express-session
- jQuery
- path - installed by default can omit from package.json
- node - uninstall this and create a new top level key like this:
```js
"engines": {
    "node":"^14.16.0",
    "npm":"^7.6.3",
}
```

This specifies the node version needed in the install, 15 is the edge version of node and I think we want to use the LTS version